packages/fx-core/tests/component/coordinator/coordinator.create.test.ts

```diff
@@ -599,6 +599,47 @@ describe("coordinator create", () => {
     const res = await coordinator.create(v3ctx, inputs);
     assert.isTrue(res.isOk());
   });
+
+  it("create non-sso tab from .NET 8", async () => {
+    const generator = sandbox.stub(Generator, "generateTemplate").resolves(ok(undefined));
+
+    // non-sso tab
+    const inputs: Inputs = {
+      platform: Platform.VS,
+      folder: ".",
+      [QuestionNames.AppName]: randomAppName(),
+      [QuestionNames.ProgrammingLanguage]: "csharp",
+      [QuestionNames.SafeProjectName]: "safeprojectname",
+      ["targetFramework"]: "net8.0",
+      [QuestionNames.ProjectType]: ProjectTypeOptions.tab().id,
+      [QuestionNames.Capabilities]: CapabilityOptions.nonSsoTab().id,
+    };
+    const fxCore = new FxCore(tools);
+    const res = await fxCore.createProject(inputs);
+
+    assert.isTrue(res.isOk());
+    assert.equal(generator.args[0][2], TemplateNames.TabSSR);
+  });
+
+  it("create sso tab from .NET 8", async () => {
```

Comment:
Better to add ut for creating tab from .NET 6.0 or .NET 7.0 for completeness.

Reply:
Well, I'll add ut.